src/Sum/README.md

```diff
@@ -0,0 +1,147 @@
+# Sum
+```haskell
+Sum Number
+```
+`Sum` is a `Monoid` that will combine two numbers using Addition.
```

Comment:
> Sum` is a `Monoid` that will combine two numbers using Addition.

We should stick with the following:

`Sum` is a `Monoid` that will combine (2) `Number`s under addition.